platform/binance/api_test.go

```diff
@@ -459,6 +506,22 @@ func testNormalizeToken(t *testing.T, _test *testToken) {
 	})
 }
 
+func testNormalizeTokenWithNotOk(t *testing.T, _test *testToken) {
```

Comment:
Improve the test structure to support error verification instead create a new method only to handle error cases

Reply:
Done